internal/util/state/metrics.go

```diff
@@ -66,6 +66,9 @@ func (mc *metricsCollector) Collect(ch chan<- prometheus.Metric) {
 		constLabels["telemetry"] = "undecided"
 	case *s.Telemetry:
 		constLabels["telemetry"] = "enabled"
+		if s.LatestVersion != v.Version {
+			constLabels["latest_version_available"] = s.LatestVersion
```

Comment:
Ideally, that would be nice (and seems to be possible) to cover it with a unit test.
However, with out current tests I don't see a simple way to do it, so let's create a separate issue to discuss if/how/what we can test here.

Reply:
[done](https://github.com/FerretDB/FerretDB/issues/1751)